sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -131,6 +132,7 @@ The following section provides several code snippets using the `client` [created
 ### Async examples
 * [Detect Language Asynchronously](#detect-language-asynchronously)
 * [Recognize Entities Asyncronously](#recognize-entities-asynchronously)
+* [Recognize Healthcare Entities Asyncronously](#recognize-entities-asynchronously)
```

Comment:
apologies for not explaining this later. We want the main scenarios be present in the Sync examples section.
The Async examples is just so the user knows that we also offer async methods.
Could you move the healthcare sample to the Sync section?